apps/web/app/api/epub/route.ts

```diff
@@ -0,0 +1,65 @@
+import { inArray } from "drizzle-orm";
+import epub, { Chapter } from "epub-gen-memory";
+
+import { db } from "@hoarder/db";
+import { bookmarkLinks } from "@hoarder/db/schema";
+
+export async function GET(request: Request) {
+  const { searchParams } = new URL(request.url);
+  const assetIds = searchParams.getAll("assetId");
+
+  if (!assetIds || assetIds.length === 0) {
+    return new Response("", {
+      status: 404,
+    });
+  }
+
+  const bookmarkInformation = await db
+    .select({
+      title: bookmarkLinks.title,
+      htmlContent: bookmarkLinks.htmlContent,
+    })
+    .from(bookmarkLinks)
+    .where(inArray(bookmarkLinks.id, assetIds));
```

Comment:
This is missing privacy checks and would allow anyone to download bookmarks they don't own